_posts/research/subteams/2021-03-16-control-artificial-skin.md

```diff
@@ -65,6 +65,28 @@ Future work in this area will focus on gentle continuous contact interactions, l
 
 {% include image.html url="research/roboskin/sensor_units.jpg" max-width="40%" description="Anticipating contact with whole-body sensing." %}
 
+
+
+## 1.3 Motion planning and trajectory optimization for contact-rich manipulation
```

Comment:
I think we should place these sections in reverse chronological order. Therefore, I would put this one as section 1.1 as it's the most recent.

Reply:
Looking at how nice the below is, I would put it first :-)

Reply:
Updated as section 1.1